src/test/resources/configuration/datasources/mysql-testcontainer.yml

```diff
@@ -1,3 +1,3 @@
-url: "jdbc:tc:mysql:5.6.23:///openmrs?TC_INITSCRIPT=mysql-testcontainer-init.sql"
+url: "jdbc:tc:mysql:5.6.23:///openmrs?TC_INITSCRIPT=mysql-testcontainer-init.sql&TC_DAEMON=true"
```

Comment:
This was needed to allow multiple SQL connections to manipulate the DB in a given test, and for the changes done in these to be visible to one another.  Incidentally, it speeds up the overall test suite execution (and thus the overall build) massively.